cdc_fuse_fs/mock_config_stream_client.h

```diff
@@ -0,0 +1,29 @@
+#ifndef CDC_FUSE_FS_MOCK_CONFIG_STREAM_CLIENT_H_
+#define CDC_FUSE_FS_MOCK_CONFIG_STREAM_CLIENT_H_
+
+#include "cdc_fuse_fs/config_stream_client.h"
+
+namespace cdc_ft {
+
+// Mock ConfigStreamClient implementation, used for testing only.
+class MockConfigStreamClient : public ConfigStreamClient {
+ public:
+  // Returns the list of relative file paths to assets that have been
+  // prioritized via ProcessAssets().
+  const std::vector<std::string>& PrioritizedAssets() const;
```

Comment:
Are these two used at all? I don't see an implementation.

Reply:
They are not, you are right. Removed.